test/index.ts

```diff
@@ -2,9 +2,14 @@ import * as dns from 'node:dns';
 dns.setDefaultResultOrder('ipv4first');
 
 import { describe, expect, test } from '@jest/globals';
+import { execSync } from 'child_process';
```

Comment:
Why are we using execSync?

Reply:
I tried using execSyncPretty but it seems like I ultimately needed {encoding: 'utf-8'}. we can add that to execSyncPretty if you want, but I wasn't sure if that would be worth it. If it is we can decide if we just want it to always be that or if we want to add another parameter or convert stdio to an options object like in execSync

Reply:
I have my reasons, but let's not worry about that, I'll just switch it to execSyncPretty